peachjam/forms.py

```diff
@@ -228,10 +237,19 @@ def send_email(self):
             context=context,
         )
 
+        app_name = settings.PEACHJAM["APP_NAME"]
+
         subject = _("Document problem reported on %(app_name)s") % {
-            "app_name": settings.PEACHJAM["APP_NAME"]
+            "app_name": app_name
         }
 
+        # fetch admin emails
+        site_admin_emails = pj_settings().admin_emails.split()
+        if site_admin_emails:
+            # list of (app_name, email) tuples
+            app_admin_emails = [(app_name, email) for email in site_admin_emails]
+            settings.ADMINS.extend(app_admin_emails)
```

Comment:
Don't manipulate the `ADMINS` constant -- you're going to manipulate it multiple times and it's going to go wrong when the setting in the admin area is changed.

Instead, as suggested, use `send_mail` instead and give it both the emails from `settings.ADMINS` and your new `admin_emails` as receipients.